domain/removal/state/model/machine.go

```diff
@@ -603,6 +591,111 @@ func (st *State) removeBasicMachineData(ctx context.Context, tx *sqlair.TX, mUUI
 	return nil
 }
 
+func (st *State) removeNetNode(ctx context.Context, tx *sqlair.TX, netNodeUUID string) error {
+	type node entityUUID
```

Comment:
I've kept this type but it's a little confusing in this function since we use a mixture of `entityUUID` when we're querying for the lld and then this `node` type when we're querying for the net node.
Lmk if I should just change it to `entityUUID`.